lk2nd/device/dts/msm8952/msm8956-sony-xperia-loire.dts

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8956 0x10001>;
+	qcom,board-id= <QCOM_BOARD_ID_MTP 0>;
```

Comment:
```suggestion
	qcom,board-id = <QCOM_BOARD_ID_MTP 0>;
```

also not sure if we would want to name the file with whatever is generic name for this mtp board

Reply:
I went with the same name used in Linux mainline (except it's a dtsi there).